packages/jest-config/src/parseShardPair.ts

```diff
@@ -0,0 +1,47 @@
+export interface ShardPair {
+  shardCount: number;
+  shardIndex: number;
+}
+
+export const parseShardPair = (pair: string): ShardPair => {
+  const parseResult = parseShardPairResult(pair);
+
+  if (parseResult instanceof Error) {
```

Comment:
when is an error `return`ed? this looks like dead code to me, only `throw` is used in `parseShardPairResult`?

Reply:
Removed